proxy/source/markdown_parser.py

```diff
@@ -9,7 +9,7 @@
 
 # The supported tags are:
 # - URLs
-# - Headers (atx-style, parses to a single level)
+# - Headers (span blocks with font-size, parses up to 4 levels)
```

Comment:
nit: we should keep `atx-style` in the comment since we don't support Setex style.